### PR TITLE
completed ticket 6

### PR DESCRIPTION
**Pull request by DanielCouperthwaite**
Hi, 

I have completed ticket number 6! 

Please let me know about any changes I need to make, thanks!

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -78,4 +79,45 @@ describe("/api/articles/:article_id", () => {
...
+describe("/api/articles/:article_id/comments", () => {
+  test("GET - status 200 - Returns a comment with a valid article id", () => {
+    return request(app)
+    .get("/api/articles/9/comments")
+    .expect(200)
+    .then((res) => {
+      expect(res.body.comments).toBeSortedBy('created_at', {
+        descending: true
+      }),
+      res.body.comments.forEach((comment) => {
```
Comment (yalicee) [inline __tests__/app.test.js:95]: check for length of array

File: __tests__/app.test.js [hunk 2]
```diff
@@ -78,4 +79,45 @@ describe("/api/articles/:article_id", () => {
...
+            article_id: expect.any(Number),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String)
+          })
+        )
+      })
+    })
+  })
```
Comment (yalicee) [inline __tests__/app.test.js:108]: need a 200 test for no comments on an article

**Review by yalicee — CHANGES_REQUESTED** [review]
Just a few changes please :)

File: __tests__/app.test.js (hunk 2 above)
Comment (DanielCouperthwaite) [reply __tests__/app.test.js:108]: Hi, thanks for the feedback! Could you just explain this one a little more please?

**Pull request merged**